docs/documentation/pages_en/quickstart.md

```diff
@@ -320,13 +329,23 @@ Or use one of the following instructions to set up the local Kubernetes cluster
    }
    ```
 
+   If there is no such file in the directory, you need to create it and insert the above lines into it. Note that you need superuser (root) privileges to access and modify files in the `/etc` directory.
+
    Restart Docker:
    
    ```shell
    sudo systemctl restart docker
    ```
 
-6. Allow usage of Container Registry without TLS for werf:
+   Then start minikube again:
```

Comment:
Shouldn't it be stopped first?

Reply:
Was stopped by `systemctl restart docker`.

Reply:
It stops when Docker restarts in the previous paragraphs. I have caught an error of an unstarted cluster twice. If it is not started, then a message about an unstarted cluster will be added to `/etc/hosts` instead of the ip address in the next step.